src/processing/spatial_operations.jl

```diff
@@ -1,38 +1,39 @@
 
 """
     simplify_exclusions!(
-    exclusions::DataFrame;
-    min_area::Real=50,
-    simplify_tol::Real=2
+        exclusions::DataFrame;
+        min_area::Float64=3E-7,
+        simplify_tol::Float64=1E-4
     )::DataFrame
 
-Simplify exclusions by removing small polygons, and simplifying geometry.
+Removing small polygons and simplify geometry based on respective tolerance values.
 
 # Arguments
-- `exclusions::DataFrame`: The DataFrame containing exclusion zones.
-- `min_area::Real`: The minimum area for exclusion polygons. Default = 50
-- `simplify_tol::Real`: The tolerance value for simplifying the exclusion polygons, i.e., larger tol = more aggressive simplification. Default = 2
+- `exclusions`: The DataFrame containing exclusion zones.
+- `min_area`: The minimum area for exclusion polygons.
+- `simplify_tol`: The tolerance value for simplifying the exclusion polygons,
+    i.e., larger tol = more aggressive simplification.
```

Comment:
Use backslash to force a new line. Otherwise it gets rendered strangely.

```suggestion
- `simplify_tol`: The tolerance value for simplifying the exclusion polygons, \
    i.e., larger tol = more aggressive simplification.
```

Reply:
`\n`